libcxx/include/cmath

```diff
@@ -610,6 +619,161 @@ _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR bool __constexpr_isfinite(_A1 __lcpp_x)
   return __builtin_isfinite(__lcpp_x);
 }
 
+_LIBCPP_CONSTEXPR inline _LIBCPP_HIDE_FROM_ABI float __constexpr_copysign(float __x, float __y) _NOEXCEPT {
```

Comment:
This code looks like it's been accidentally re-added.

Reply:
Fixed in 6a6fce8a1673355fc72f6ddfbbc1fa3d9d2d9d87.